<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotionUi</class>
 <widget class="QDialog" name="MotionUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SceneMotion</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Video source :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSource">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtOut">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>351</width>
     <height>171</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Output :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btStart">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Steps :</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="handleWidth">
    <number>10</number>
   </property>
   <widget class="QCheckBox" name="chkAffine">
    <property name="text">
     <string>1. Affine</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkwflow">
    <property name="text">
     <string>2. w-flow</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkProfiling">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>3. Motion Estimation</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Spline :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Clusters :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>MaxDist :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMaxDist">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MinClustSize :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Bkg : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMinClustSize">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMinValue">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Others :</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkWriteMotion">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Write motion.clust</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdWflow">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>wflow</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdAfflow">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>afflow</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdBoth">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>both</string>
   </property>
  </widget>
  <widget class="QSlider" name="slSmooth">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>100</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoothing :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSmooth">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>frames</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkCorrectErrors">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Correct possible errors with values &gt; </string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbCorrection">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>900</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkAdaptive">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Use adaptive clustering</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>slSmooth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lblSmooth</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
